### PR TITLE
Add credits section

### LINKED ISSUE #100: Credits Section
image

This credit section's design is not in the figma link, but just one of the personal references I have. Can also to the one on Fintech Month for implementation reference if needed.

### PR DESCRIPTION
**Pull request by zzhikai**
#100 
Referenced FTS-Month Implementation
Issue Example:
image

Current Implementation: 
Differences: 

- Did not include top "Partners" title.
- Background colour not changed

_Desktop View_
Screenshot 2022-01-29 at 8 36 33 PM

_Mobile View_
Screenshot 2022-02-02 at 7 09 58 AM

### CODE REVIEW

File: src/components/Credits/index.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    flexDirection: 'column',
+    marginTop: 20,
+    [theme.breakpoints.down('xs')]: {
+      marginTop: 32,
+    },
+  },
+  sectionHeader: {
+    fontWeight: 600,
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 20,
```
Comment (jerrenss) [inline src/components/Credits/index.jsx]: Can we increase the size of the headers in mobile? Think 32px will be good

File: src/components/Credits/index.jsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          Partners
+        </Typography>
+        <Grid
+          container
+          className={classes.rowWrapper}
+          spacing={4}
+          style={{ maxWidth: '900px' }}
+        >
+          {partnersData.map((sponsor, index) => (
+            <Grid item xs={12} sm={3} key={index}>
```
Comment (jerrenss) [inline src/components/Credits/index.jsx]: For partners can we do xs={6}? So we show two columns instead of one, just like in FTS Month

**Review by jerrenss — COMMENTED** [review]
Looks good! Just small changes required.

File: src/components/Credits/index.jsx (hunk 1 above)
Comment (zzhikai) [reply src/components/Credits/index.jsx]: 22b8520942bac69d7c64469c4f12f8620d9d7ae6

File: src/components/Credits/index.jsx (hunk 2 above)
Comment (zzhikai) [reply src/components/Credits/index.jsx]: 22b8520942bac69d7c64469c4f12f8620d9d7ae6

**Review by jerrenss — APPROVED** [review]
Thanks!

**Pull request merged**